hwilib/commands.py

```diff
@@ -136,40 +134,79 @@ def signmessage(args, client):
     return client.sign_message(args.message, args.path)
 
 def getkeypool(args, client):
-    # args[0]: path base (e.g. m/44'/0'/0')
-    # args[1]; start index (e.g. 0)
-    # args[2]: end index (e.g. 1000)
-    # args[3]: internal (e.g. False)
-    path_base = args.path_base
+    # args[0]; start index (e.g. 0)
+    # args[1]: end index (e.g. 1000)
+    path = args.path
     start = args.start
     end = args.end
     internal = args.internal
     keypool = args.keypool
+    account = args.account or 0
 
     master_xpub = client.get_pubkey_at_path('m/0h')['xpub']
     master_fpr = get_xpub_fingerprint_as_id(master_xpub)
 
+    if not path:
+      # Master key:
+      path = "m/"
+
+      # Purpose
+      if args.bech32 == True:
+        path += "84'/"
+      elif args.p2sh_p2wpkh == True:
+        path += "49'/"
+      else:
+        path += "44'/"
+
+      # Coin type
```

Comment:
which BIP is this?

Reply:
Yes, there's a list of coin types in BIP 44: https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#registered-coin-types

That includes mainnet as type 0 and testnet as type 1.